rest_framework/authentication.py

```diff
@@ -150,6 +150,7 @@ class TokenAuthentication(BaseAuthentication):
         Authorization: Token 401f7ac837da42b97f613d789819ff93537bee6a
     """
 
+    keyword = 'token'
```

Comment:
I'd suggest we use the proper casing here, and instead do `.lower()` in the comparison against the auth header.